### ISSUE TITLE
Request to add prompt field for inline-choice question type.

### ISSUE THREAD

**Issue by danish-tr** [pie-framework/pie-elements#524]
I've found the [commit](https://github.com/pie-framework/pie-elements/commit/b91827ffa0c4a9e24d97d8dece44c12e59f0e3a5) where prompt field for inline-choice question type has been removed. 

![inline-choice](https://user-images.githubusercontent.com/43806379/109793817-f545c280-7c36-11eb-8979-d20d372cb6ce.png)

Is there any reason to remove the prompt field? 

Currently we need this prompt field for inline-choice type. Is there any way to add it again?

**Comment by andreeapescar** [comment]
If you mean inline-dropdown, @danish-tr , we still have prompt support: https://github.com/pie-framework/pie-elements/blob/develop/packages/inline-dropdown/configure/src/main.jsx

            {promptEnabled && (
              <InputContainer
                label={prompt.label}
                className={classes.promptHolder}
              >
                <EditableHtml
                  className={classes.prompt}
                  markup={model.prompt}
                  onChange={this.onPromptChanged}
                  imageSupport={imageSupport}
                  nonEmpty={false}
                  disableUnderline
                />
              </InputContainer>
            )}

**Comment by danish-tr** [comment]
@andreeapescar Yes, your are right. **inline-dropdown** have prompt field. My question is for **inline-choice**. These two are different question types. 

Is there any other way to add prompt field for **inline-choice**?

So far,  I clone [pie-framework](https://github.com/pie-framework/pie-elements) repository and I've made changes to add prompt field for **inline-choice** inside old-packages directory. Now I'm unable to verify my changes due to scripts inside package.json e.g. yarn build, yarn test. 

Secondly, I follow [this ](https://github.com/pie-framework/pie-elements#running) to run cloned repo on my machine and so far I didn't succeeded. 

Is there any clear docs to apply running process for pie-elements repo?

**Comment by danish-tr** [comment]
@andreeapescar Any update regarding **inline-choice** type that I've mentioned?
